andela/elven-ah-client
# #160056177 set-up-sass-processing-on-the-client

#### What does this PR do?
Set up sass processing on the client
#### Description of Task to be completed?
- install sass-loader and node-sass
- configure sass on webpack-config.js
#### How should this be manually tested?
- create a .scss file and write some styling, 
- import the created file in a .js file
- run npm start
- You should see your style being effected without any compilation error
#### What are the relevant pivotal tracker stories?
[#160056177](https://www.pivotaltracker.com/story/show/160056177)
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/33626790/44905514-2a8d9480-ad0a-11e8-9add-4e5cafe75d2d.png)
